### ISSUE TITLE
Checkboxes in list view should show over bottom right of filetype icon / thumbnail

### ISSUE THREAD

**Issue by jancborchardt** [owncloud/android#1433]
Currently it's very disconnected having them on the right. Checkboxes are usually on the left, also ref our web interface.

Same for the grid view btw, bottom right of the thumbnail is where the checkboxes should always go. :)

**Comment by AndyScherzinger** [comment]
So where should the star go then? Just asking since atm the star icon is places on the bottom right of the icon/thumbnail so that would have to move somewhere else then.

**Comment by jancborchardt** [comment]
Damn, that's right. The star should go on the top right of the thumbnail. Any other icons/ badges on the bottom left. Does that sound reasonable?

**Comment by AndyScherzinger** [comment]
That would mean the status (downloaded, downloading, failed etc.) which is currently on the top right would have to move to the bottom left. So then you would have 3 mini icons around your thumbnail/icon one in each corner except for the top left. 

We do have the share icon on the right of the whole list item:
* chain for a share by link
* share icon for user/group share

We could also move the star icon (fav) to the right and place it above the icon. 

Just an idea. The problem for me right know is that the line item is able to display 4 different kind of information (besides name, size etc.) via icons.

So tl;dr yes that is possible but seems to add visual complexity and more information, but I honestly don't have better idea. Except for the one mentioned, that is how gmail does it, but this isn't a mail client...

**Comment by cornzy** [comment]
@jancborchardt I prefere having the checkbox on the upper left corner (like in other android apps like Google Foto). In Multi-Select you normally go through the grid from upper left to lower right. If you use your right hand (especially your thumb) you always cover the lower right corner, the upper left is most visible. Also with the left hand the checkbox in upper-left corner is well visible.

In the list view I do not need a checkbox at all, marking the line item with a darker color is enough - also compare other android (material designed) apps (like hangout). If you realy want the checkbox it should be on the very left. On the right it again it is covered by your finger and checkbox always have to be on the left (compare any style guide). Additionally it is realy ugly combined with the share icon because the checkbox moves left and breaks the line of checkboxes.

**Comment by jancborchardt** [comment]
True @cornzy, good points. What do you think about that @tobiasKaminsky @AndyScherzinger @davivel @masensio? No checkboxes in list view (only row highlight) and checkboxes on top left in grid view.

**Comment by AndyScherzinger** [comment]
Hmm, tricky one.... 

What I see in many *list* based interfaces which are material styled they add a check mark by replacing the icon work on the left and changing the background color of the list item. Hangout doesn't do this, it only changes the background color (as mentioned by @cornzy) While other apps do, for Google e.g. GMail, Inbox and Drive, for other non Google app Amaze would be an example.

As for grid based interfaces with material style the also usually add a check mark. The position and size differs between app even between material themed Google apps. There is also no guideline with material design on this level. Only thing about multi selection: https://www.google.com/design/spec/patterns/selection.html#selection-item-selection - only covers behavior, not visuals.

So we could do this is a two step way:
1 - implement it in the list with just the background coloring for lists and the check boxes for grids
2 - implement the left side icon replacing check mark when item is selected

**Comment by tobiasKaminsky** [comment]
I have no preferences...You are the design gurus. 
Whatever you decide I will fine with it :+1:

**Comment by AndyScherzinger** [comment]
@jancborchardt can you look at the comments for the multi select? Problem is the bottom left is already taken with the status icon for the file/folder. This is an information not available in the web frontend for abvious reasons ;) So for now we "just" put it to the top/left since that is the only place available and grid items file file name also look different, so top left seems the most practical one. I posted some screenshots there https://github.com/owncloud/android/issues/899#issuecomment-195948600

**Comment by jesmrec** [comment]
Changes not expected ftm
